src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+
+        Machine bulldozer = new Bulldozer();
+
+        Machine truck = new Truck();
+
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
 
```

Comment:
17 line can also be removed

Reply:
I can't remove last line, because I will get such mistake
C:\Users\Acer\IdeaProjects\jv-oop\src\main\java\core\basesyntax\MainApp.java:1: File does not end with a newline. [NewlineAtEndOfFile]
Audit done.